js/flickity.js

```diff
@@ -764,22 +775,25 @@ proto.watchCSS = function() {
 
 // go previous/next if left/right keys pressed
 proto.onkeydown = function( event ) {
-  // only work if element is in focus
-  if ( !this.options.accessibility ||
-    ( document.activeElement && document.activeElement != this.element ) ) {
-    return;
-  }
-
-  if ( event.keyCode == 37 ) {
-    // go left
-    var leftMethod = this.options.rightToLeft ? 'next' : 'previous';
-    this.uiChange();
-    this[ leftMethod ]();
-  } else if ( event.keyCode == 39 ) {
-    // go right
-    var rightMethod = this.options.rightToLeft ? 'previous' : 'next';
-    this.uiChange();
-    this[ rightMethod ]();
+  // only work if child of carousel is focused
+  if ( this.options.accessibility && this.element.contains(document.activeElement) ) {
```

Comment:
I like how this works by looking for the focused element within the Flickity container, but I'm worried that its to general. For example, you could have a text input that needs to have separate keyboard controls from Flickity's behavior.